libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -679,10 +683,12 @@ static CK_RV prvRsaKeyAttParse( const CK_ATTRIBUTE * pxAttribute,
                                 const mbedtls_pk_context * pxMbedContext )
 {
     CK_RV xResult = CKR_OK;
-    int32_t lMbedReturn = 0;
+    int32_t lMbedTLSResult = 0;
     CK_BBOOL xBool;
     mbedtls_rsa_context * pxRsaContext = ( mbedtls_rsa_context * ) pxMbedContext->pk_ctx;
```

Comment:
`configASSERT` for `pxMbedContext` before de-referencing it?

Reply:
Oops, moving this